src/main/scala/org/coroutines/Coroutine.scala

```diff
@@ -172,10 +172,26 @@ object Coroutine {
       $yield
     }
 
+    /** Returns whether or not the coroutine yielded a value. This value can be
```

Comment:
Please separate the 2nd sentence into its own line, delimited from the first line with one empty line.